webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -1344,6 +1344,43 @@
                 rdfs:label  "Transkription von"@de ,
                             "Transcription of"@en .
 
+### ###########################################
+### beol:belongsToRegion
+
+    :belongsToRegion rdf:type owl:ObjectProperty ;
+
+           rdfs:subPropertyOf knora-base:hasLinkTo ;
+
+           rdfs:comment "Transkription von"@de ,
+                        "Transcription of"@en ;
+
+           rdfs:label  "Transkription von"@de ,
+                       "Transcription of"@en ;
+
+           knora-base:subjectClassConstraint :transcription ;
+
+           knora-base:objectClassConstraint knora-base:Region ;
```

Comment:
@benjamingeer With the object class constraint of `knora-base:Region`, I got an error in the integration test "should create a sample BEOL letter" in `org/knora/webapi/e2e/v1/KnoraSipiIntegrationV1ITSpec.scala` which attempts to create a letter using the bulk import:

> RESPONSE: {"status":11,"error":"org.knora.webapi.BadRequestException: XML import did not pass XML schema validation: org.xml.sax.SAXParseException; lineNumber: 351; columnNumber: 132; src-resolve: Cannot resolve the name 'knoraXmlImport:Region_linkValueType' to a(n) 'type definition' component."} (ITKnoraLiveSpec.scala:112)

Then I changed the object class constraint to `knora-base:Resource` and it worked fine. 

Does the error message say that it cannot find the definition for the object class constraint `knora-base:Region` when doing the schema validation?

Reply:
this happened in 
https://github.com/dhlab-basel/Knora/pull/890/commits/cfe6ee1ec82ad9be2f6a9ae32fedde10f2cacc19

Reply:
This must be the same problem as in #1171. You can fix it in the same way. Let me know if you need help.

Reply:
The it test passes locally now. I can wait for the fix and change it when #1171 is fixed.

Reply:
#1171 is a PR that was already merged to fix a similar problem. I’m saying that if you look at that PR, you’ll see how to fix the problem you’re getting here.

Reply:
Ok, I would simply have to add `knora-base:Region` to `webapi/src/main/resources/knoraXmlImport.xsd` if I want to use it as an object class constraint (in analogy what was done for `knora-base:Representation`). I would rather do that in a small separate PR.

Reply:
No problem to do it in a separate PR.

Reply:
in progress in https://github.com/dhlab-basel/Knora/pull/1225

Reply:
reapplied in 
https://github.com/dhlab-basel/Knora/pull/890/commits/7bc90dba0d97da1fd0e4ce45e5956e934fa7dfef